File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt for height
    int n = 0;
    while (n < 1 || n > 8)
    {
        n = get_int("Height? ");
    }

    // Print hashes
    for (int j = 1; j <= n; j++)
    {
        for (int i = 0; i < (n - j); i++)
        {
            printf(" ");
        }
        for (int i = 0; i < j; i++)
        {
            printf("#");
        }
        printf("  ");
        for (int i = 0; i < j; i++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    height = get_height()

    for i in range(1, height + 1):
        print(" " * (height - i), "#" * i, "  ", "#" * i, sep="")


def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n <= 8:
            return n


main()
